tg_bot.py

```diff
@@ -250,6 +263,27 @@ def print_unsubscribe_button(msg):
         reply_markup=InlineKeyboardMarkup(custom_keyboard, one_time_keyboard=True))
 
 
+def print_subscription_status(update, msg, context): 
+    # include the parameters
+    """
+    Prints current subscription status
+    """
+    user_id = str(update.effective_user.id)
+
+    # format date 
+    subscription_limit = scheduled_jobs['limit_subs'].date()
```

Comment:
Replace to `subscription_limit  = scheduled_jobs[user_id].date() + datetime.timedelta(days=7)`

Reply:
Done.